packages/carbon-react/.storybook/styles.scss

```diff
@@ -22,3 +46,16 @@ body {
 [data-carbon-theme='g100'] {
   @include styles.theme(styles.$g100);
 }
+
+html[lang='en'] body {
+  font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
+}
+
+html[lang='ar'] body {
+  font-family: 'IBM Plex Sans Arabic', 'Helvetica Neue', Arial, sans-serif;
```

Comment:
@vpicone do we have font stacks that we could use for sans/arabic/etc? I know we define some over in: https://github.com/carbon-design-system/carbon/blob/main/packages/type/scss/_font-family.scss#L13 but was curious if you think this should be in the `@ibm/plex` package instead

Reply:
BoldMonday doesn't have any recommended fallback fonts outside of Latin. Might be worth exploring what other folks like Noto are doing.

Reply:
@vpicone I think the `font-family` rules for Mono/Sans/Serif are mentioned in the README of the project: https://github.com/IBM/plex#web-usage

Just was asking about codifying those in the sass itself since it seemed like they are a part of using the package on the web 👍

Reply:
Totally, I was thinking about the non-english languages that aren't supported by fallback/web safe stack.

Reply:
@vpicone would we end up doing something like:

```css
system-ui, -apple-system, BlinkMacSystemFont, '.SFNSText-Regular', sans-serif
```

? Seems to work for FB across the different languages, falling back to specific local files ultimately (I think)

Reply:
Definitely, there's also some Web Safe fonts for each language that might be more appropriate matches, this comes into play when the font-display is swap and you get the flash of System text. But using the systems font is simple and safe.

Reply:
I'm down to just use system fonts as fallbacks! I just copied what we had in the readme for now so the linter would stop yelling at me for not declaring any fallbacks. Whatever we decide, we can then make it reusable as suggested in https://github.com/carbon-design-system/carbon/pull/8726#discussion_r637062234